ui/StatusQ/include/StatusQ/modelutilsinternal.h

```diff
@@ -26,6 +26,12 @@ class ModelUtilsInternal : public QObject
 
     Q_INVOKABLE bool contains(QAbstractItemModel *model, const QString &roleName, const QVariant &value, int mode = Qt::CaseSensitive) const;
 
+    //!< traverse the permissions @p model, and look for unique token keys recursively under holdingsListModel->key
```

Comment:
My only concern here, but also discussed in the teams sync meeting, is that these 2 methods are specific for the status app and I would say they should not be part of `statusq`. However, if the alternative implies a lot of extra work I would suggest to create a technical debt task to address it later.

Reply:
Yeah, I think we agreed to keep them in StatusQ; question really is whether they belong to ModelUtils? Open to suggestions, perhaps create a dedicated `PermissionsUtils`?

Reply:
For me it sound reasonable to have a `PermissionsUtils`.